scripts/setup.sh

```diff
@@ -28,5 +39,5 @@ fi
 echo 'Updating .env.shared file (for shared configuration)...'
 aws s3 cp s3://artsy-citadel/dev/.env.metaphysics .env.shared || return
```

Comment:
This is the last `|| return` so we may as well remove this too?
```suggestion
aws s3 cp s3://artsy-citadel/dev/.env.metaphysics .env.shared || 'Unable to download shared configuration, ensure you have S3 access!'
```